test/test-loops-2.js

```diff
@@ -0,0 +1,45 @@
+function multiplicationTable() {
```

Comment:
seems duplicated file

make sure you do not have your functions redefined in test files
you should require them